datahub-graphql-core/src/main/resources/entity.graphql

```diff
@@ -1331,6 +1331,15 @@ type LineageRelationship {
     """
     isManual: Boolean
 
+    """
+    The paths traversed for this relationship
+    """
+    paths: [EntityPath]
+
+    """
+    Any in-between via nodes for this relationship
+    """
+    via: [String]
```

Comment:
this is the first time we are introducing via as a concept in our APIs. Should we?